src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -29,9 +29,9 @@ To create your App Store listing,
 
 include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
 
-. Click *New App Template*.
+. In your list of templates/listings, click *New App Template*.
 
-. Give the app template a name.
+. Give the app template/listing a name.
```

Comment:
```suggestion
. Give your listing a name.
```
Suggest removing the slashes. This is just one way to do it.

Reply:
Fixed.